curvine-client/src/unified/unified_filesystem.rs

```diff
@@ -315,14 +315,24 @@ impl FileSystem<UnifiedWriter, UnifiedReader, ClusterConf> for UnifiedFileSystem
     async fn get_status(&self, path: &Path) -> FsResult<FileStatus> {
         match self.get_mount(path).await? {
             None => self.cv.get_status(path).await,
-            Some((ufs_path, mount)) => mount.ufs.get_status(&ufs_path).await,
+            Some((ufs_path, mount)) => {
+                if mount.info.cv_path == path.path() {
+                    return self.cv.get_status(path).await;
+                }
+                mount.ufs.get_status(&ufs_path).await
+            }
         }
     }
 
     async fn get_status_bytes(&self, path: &Path) -> FsResult<BytesMut> {
         match self.get_mount(path).await? {
             None => self.cv.get_status_bytes(path).await,
-            Some((ufs_path, mount)) => mount.ufs.get_status_bytes(&ufs_path).await,
+            Some((ufs_path, mount)) => {
```

Comment:
This destroys the semantics of transparent transmission. Even if you want to access the root directory of the mount point, you still need to access ufs directly.